TicTacToe/TicTacToe.js

```diff
@@ -1,105 +1,95 @@
 class Game {
-    moves = 0;
-    user = 'X';
-    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
-    messages = "Game - START!";
-    complete = false;
 
+    constructor() {
+        this.reset();
+    }
+
+    // resets the board
     reset() {
+        this.n = 3;
         this.moves = 0;
         this.user = 'X';
         this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
         this.messages = "Game - START!";
         this.complete = false;
     }
 
-    getMoves() {
-        return this.moves;
-    };
-
+    // displays the board in text form
+    // don't really need this, but it's good to see
     displayBoard() {
-        let text = '';
-        for (let i = 0; i < 3; i++) {
-            for (let k = 0; k < 3; k++) {
-                text += " " + this.board[i][k];
-            }
-            text += "\n";
-        }
+        let text = this.board.join("\n").replaceAll(",", " ");
         console.log(text);
         return text;
     };
 
+    // returns any messages from the game
     getMessages() {
         return this.messages;
     }
 
+    // returns current user
     makeMove(x, y) {
         this.messages = "";
         if (this.complete) {
             this.messages = "GAME COMPLETED";
-            return;
+            return this.user;
         }
 
-        if (this.board[x][y] != '-') {
+        if (this.board[x][y] !== '-') {
             this.messages = "Invalid move! Try again!";
-            if (this.user == 'X') {
-                return 'O';
-            } else {
-                return 'X';
-            }
+            return this.user === 'X' ? 'O' : 'X';
         }
 
         this.board[x][y] = this.user;
         this.moves++;
 
         if (this.isWin(x, y)) {
-            this.messages = this.user + " is the winner!";
+            // string literal- why not just make every string like this then?
```

Comment:
Well, string interpolation is not literal anymore -- there's probably a small overhead to doing this. But it's not something to really worry about. I think it's still conventional to use quotes (either single or double, as long as you stay consistent)